package.json

```diff
@@ -16,7 +16,9 @@
 		"start": "cgb-scripts start",
 		"build": "cgb-scripts build",
 		"distclean": "rm -rf node_modules && rm -rf dist",
-		"preinstall": "npx npm-force-resolutions"
+		"preinstall": "npx npm-force-resolutions",
+	    "archive": "composer archive --file=$npm_package_name --format=zip",
+	    "release": "npm run build && npm run archive"
```

Comment:
I wonder if we could add it as part of the `build` script too similar to Sensei, so we keep consistency in the scripts between the projects while releasing. WDYT?

Reply:
Yes, that's a good idea!